src/lusty/filesystem-explorer.rb

```diff
@@ -137,6 +137,11 @@ def all_files_at_view
 
       unless @memoized_dir_contents.has_key?(view)
 
+        view_str = view.to_s
+        if view_str.start_with?("scp://")
```

Comment:
Is it really more clear than a `.start_with?` check? Using a `is_scp_path` is not simpler, and a reader needs to follow the function to know exactly what a "scp path" is like.

If we add more protocols support, or more complex checks, it'll be worth it. Otherwise I prefer one less layer of indirection.